src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java

```diff
@@ -279,4 +267,22 @@ private void createIconResource() throws InterruptedException, IOException {
             }
         }
     }
+
+    // During development if user changes the application icon, the same is not reflected immediately in Explorer.
+    // To fix this, a cache clearance of the Windows explorer is required.
+    private void clearExplorerCache() throws IOException, InterruptedException {
+        ProcessRunner clearCache = new ProcessRunner("ie4uinit");
+        clearCache.addArg(findCacheFlag());
+        clearCache.runProcess("Clear Explorer cache");
+    }
+
+    // For Windows build > 10000, use `ie4uinit.exe -show`
+    // For Windows build < 10000, use `ie4uinit.exe -ClearIconCache`
+    private String findCacheFlag() throws IOException, InterruptedException {
+        ProcessRunner buildNumberProcess = new ProcessRunner("powershell.exe", "(Get-ComputerInfo).OsBuildNumber");
```

Comment:
Will this work in all Windows versions? Tried on Windows 7 and the command `Get-ComputerInfo` is not recognised.
Maybe you can put it under try-catch, and if it fails we can assume it is old enough to be < 10000?

Reply:
What exception is thrown on Windows 7?

Reply:
The [`Get-ComputerInfo` cmdlet](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-computerinfo?view=powershell-7.2#description) was introduced in PowerShell 5.1, so it's possible that have an older version?